src/workload.rs

```diff
@@ -94,6 +94,8 @@ pub struct Workload {
     #[serde(default)]
     pub namespace: String,
     #[serde(default)]
+    pub trust_domain: String,
```

Comment:
may need to display `trust_domain` in `fmt::Display for Workload` if add this field.